deepchem/molnet/load_function/uspto_datasets.py

```diff
@@ -67,7 +79,8 @@ def create_dataset(self) -> Dataset:
 
 
 def load_uspto(
-    featurizer: Union[dc.feat.Featurizer, str] = dc.feat.DummyFeaturizer(),
+    featurizer: Union[dc.feat.Featurizer, str] = RxnFeaturizer(
```

Comment:
Ah the issue here is the default value! This requires HuggingFace Transformer to be installed since `RxnFeaturizer` requires HuggingFace to be installed. Maybe set this by default to be a string "rxnfeaturizer" and initialize it internally in the method instead

Reply:
I tried to change it to the string args, but the checks still seem to fail.